lib/cisco_node_utils/bfd_global.rb

```diff
@@ -97,5 +210,87 @@ def fabricpath_vlan=(val)
     def default_fabricpath_vlan
       config_get_default('bfd_global', 'fabricpath_vlan')
     end
+
+    def default_interval
+      config_get_default('bfd_global', 'interval')
+    end
+
+    def default_ipv4_interval
+      config_get_default('bfd_global', 'ipv4_interval')
+    end
+
+    def default_ipv6_interval
+      config_get_default('bfd_global', 'ipv6_interval')
+    end
+
+    def default_fabricpath_interval
+      config_get_default('bfd_global', 'fabricpath_interval')
+    end
+
+    # interval is an array of interval, min_rx and multiplier
+    # CLI: bfd interval 100 min_rx 100 multiplier 25
+    def interval
+      config_get('bfd_global', 'interval', @get_args)
+    end
+
+    # ipv4_interval is an array of ipv4_interval, ipv4_min_rx and
+    # ipv4_multiplier
+    # CLI: bfd ipv4 interval 100 min_rx 100 multiplier 25
+    def ipv4_interval
+      config_get('bfd_global', 'ipv4_interval', @get_args)
+    end
+
+    # ipv6_interval is an array of ipv6_interval, ipv6_min_rx and
+    # ipv6_multiplier
+    # CLI: bfd ipv6 interval 100 min_rx 100 multiplier 25
+    def ipv6_interval
+      config_get('bfd_global', 'ipv6_interval', @get_args)
+    end
+
+    # fabricpath_interval is an array of fabricpath_interval,
+    # fabricpath_min_rx and fabricpath_multiplier
+    # CLI: bfd fabricpath interval 100 min_rx 100 multiplier 25
+    def fabricpath_interval
+      config_get('bfd_global', 'fabricpath_interval', @get_args)
+    end
+
+    # interval is an array of interval, min_rx and multiplier
+    # ex: ['100', '100', '25']
+    # CLI: bfd interval 100 min_rx 100 multiplier 25
+    def interval=(arr)
+      set_args_keys(intv: arr[0], mrx: arr[1], mult: arr[2],
```

Comment:
nit: I'm all for short variable names but they should still be meaningful, i.e. not require a comment to know what they are. These fields aren't that long so I would just keep the full name for each variable. I'm also a fan of multi-variable assignment, e.g.:

```
  interval, min_rx, multiplier = arr
  set_args_keys(interval: interval, min_rx: min_rx, multiplier: multiplier,
    ...
```

...but again these are just nits so your call.